pkg/server/server.go

```diff
@@ -34,6 +35,10 @@ func init() {
 	prometheus.MustRegister(metrics.Uptime)
 }
 
+// StartHttpServer boots up the server
```

Comment:
One last review make it 2 line

Reply:
Sorry for the late reply, will do it @shovanmaity 👍 thanks

Reply:
@rbo13 there was a misunderstanding. I wanted to say convert 5 lines of comments to 2 lines of comments. :slightly_frowning_face:

Reply:
Ooops, my bad :( will change my PR now. Thanks for clarifying